### PR TITLE
Add logging configuration info

### PR DESCRIPTION
**Pull request by vitvakatu**

### CODE REVIEW

File: src/get-started/java-binding.md [hunk 1]
```diff
@@ -712,6 +712,47 @@ service_name1 = "/path/to/service1_artifact.jar"
...
+
+See [`env_logger` documentation][env_logger-docs] for more information
+on possible configuration options.
+
+### Java Logging
+
+Java logs are produced by Java Binding internal code and can also be
+used to monitor Java services internal logic.
+
+Java logging configuration is controlled by configuration file. Example
```
Comment (bullet-tooth) [inline src/get-started/java-binding.md]: Java logging configuration is controlled by the configuration file specified by the `ejb-log-config-path` parameter.  If no file was provided, the logs are disabled. 
Exonum Java package provides an example `log4j-fallback.xml` configuration file that can be found at the installation directory.
Also, see [Application Guide][app-tutorial] for more information on
configuring the Exonum Java App.

File: src/get-started/java-binding.md [hunk 2]
```diff
@@ -712,6 +712,47 @@ service_name1 = "/path/to/service1_artifact.jar"
...
+
+Rust logs are produced by Exonum Core and can be used to monitor the
+status of the blockchain node, including information about the block
+commitment and the consensus status.
+
+Rust logs are disabled by default and controlled by the `RUST_LOG`
+environment variable. It is recommended to set `info` logging level:
+
+```bash
+export RUST_LOG=warn,exonum=info,exonum-java=info
```
[the remark this answers is not in the document]
Comment (vitvakatu) [reply src/get-started/java-binding.md]: Added more specific link to the docs (to the specific section of the docs), slightly reworded. I don't think we need explain the syntax in detail.

File: src/get-started/java-binding.md [hunk 3]
```diff
@@ -712,6 +712,49 @@ service_name1 = "/path/to/service1_artifact.jar"
...
+
+Java logging configuration is controlled by the configuration file
+specified by the `ejb-log-config-path` parameter. If no file was
+provided, the logs are disabled. Exonum Java package provides an
+example `log4j-fallback.xml` configuration file that can be found
+at the installation directory.
+Also, see [Application Guide][app-tutorial] for more information on
+configuring the Exonum Java App.
+
+Default `log4j-fallback.xml` file outputs `INFO`-level messages to
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: That contradicts 'If no file was provided, the logs are disabled'.

File: src/get-started/java-binding.md [hunk 4]
```diff
@@ -712,6 +712,49 @@ service_name1 = "/path/to/service1_artifact.jar"
...
+```
+
+Log entries go to `stderr` by default.
+
+See [`env_logger` documentation][env_logger-docs] for more information
+on possible configuration options.
+
+### Java Logging
+
+Java logs are produced by Java Binding internal code and can also be
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: ?

Logs produced by Java code (the framework and its dependencies, and the deployed services) are handled by Log4J framework. The services can use either Log4J or SLF4J logging APIs.

File: src/get-started/java-binding.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (dmitry-timofeev) [reply src/get-started/java-binding.md]: That's good

File: src/get-started/java-binding.md [hunk 5]
```diff
@@ -738,21 +738,21 @@ on possible configuration options.
...
+The services can use either [`Log4J`][log4j-home] or
+[`SLF4J`][slf4j-home] logging APIs.
 
 Java logging configuration is controlled by the configuration file
 specified by the `ejb-log-config-path` parameter. If no file was
 provided, the logs are disabled. Exonum Java package provides an
 example `log4j-fallback.xml` configuration file that can be found
-at the installation directory.
+at the installation directory. With this file `INFO`-level messages
+are output to stdout.
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: Past participle appears to be 'ouputted': https://www.oxfordlearnersdictionaries.com/definition/english/output_2

Maybe printed is better?

**Pull request merged**